Jenkinsfile

```diff
@@ -7,6 +7,9 @@ node {
       checkout scm
     }
 
+    env.RF_SETTINGS_BUCKET = 'rasterfoundry-staging-config-us-east-1'
+    env.RF_AWS_CONFIG_BUCKET = 'rasterfoundry-testing-config-us-east-1'
+    env.RF_AWS_ARTIFACTS_BUCKET = 'rasterfoundry-global-artifacts-us-east-1'
```

Comment:
Fixed in https://github.com/azavea/raster-foundry/pull/1269/commits/3dcf90a87538b444ac6fd9a9e0809ffb916e1d69